### DISCUSSION TITLE
Is serverpod ready for windows

### DISCUSSION THREAD

**Discussion by kazerdira** [serverpod/serverpod#1448] [Q&A] [answered]
I posted before, before couples of months, now i come again to test it, i installed the docker, did the commands, when i try this command 
dart bin/ main.dart 
I am getting this
 SERVERPOD version: 1.1.1, dart: 3.2.0-184.0.dev (dev) (Wed Sep 20 05:03:07 2023 -0700) on "windows_x64", time: 2023-10-11 11:56:50.283412Z mode: development, role: monolith, logging: normal, serverld: default Failed to connect to the database. Retrying in 10 seconds. PostgreSQLSeverity.error 42P01: relation "serverpod_runtime_settings" does not exist

**Comment by y3l1n4ung** [comment]
<img width="954" alt="Screenshot 2023-10-11 at 9 16 07 PM" src="https://github.com/serverpod/serverpod/assets/40236753/82a1a282-b297-4874-bee4-46caa2a80a2c">
Check your projectname_server container on docker and make sure your postgres is running

**Reply by kazerdira** [reply]
it is running and reporting in the logs that error 

2023-10-11 12:54:15.223 UTC [561] ERROR:  relation "serverpod_runtime_settings" does not exist at character 15
2023-10-11 12:54:15.223 UTC [561] STATEMENT:  SELECT * FROM serverpod_runtime_settings WHERE TRUE LIMIT 1
2023-10-11 12:54:25.577 UTC [563] ERROR:  relation "serverpod_runtime_settings" does not exist at character 15
2023-10-11 12:54:25.577 UTC [563] STATEMENT:  SELECT * FROM serverpod_runtime_settings WHERE TRUE LIMIT 1

**Comment by Isakdl** [comment, marked as answer]
You get this error when you haven't initialized the database. Run the `setup-tables.cmd` script in the root folder and you should be able to get it to run!

**Reply by kazerdira** [reply]
what do you mean by root folder 
there is three folder , the client the server and flutter ,u mean in server folder i guess,

because the command  setup-tables.cmd is not recognized , i am in windows !

**Reply by Isakdl** [reply]
Yes the server folder, sorry about the confusion!

**Reply by Isakdl** [reply]
`setup-tables.cmd` is a file in the folder that you have to execute!

win + r, type `cmd` press enter.

`cd \path\to\project\project_name_server`

`.\setup-tables.cmd`

You can also see this issue here: https://github.com/serverpod/serverpod/issues/1424
